src/main/java/bio/terra/service/configuration/ConfigurationService.java

```diff
@@ -113,6 +113,17 @@ public ConfigListModel setConfig(ConfigGroupModel groupModel) {
     return new ConfigListModel().items(priorConfigList).total(priorConfigList.size());
   }
 
+  public void setConfigParameterValue(ConfigEnum config, String newValue, String label) {
+    bio.terra.model.ConfigModel retryConfigModel =
```

Comment:
You've got some fully qualified class names here

Reply:
Also, FWIW, I think this will overwrite an entire group's worth of config.  Not sure if that ok?